doc/asciidoc/producer/configuration.adoc

```diff
@@ -32,4 +36,79 @@ all the messages sent to topics that don't exist are discarded;
 this because the `KafkaProducer.send()` method blocks the execution, as explained in https://issues.apache.org/jira/browse/KAFKA-3539[KAFKA-3539].
 You can tune the custom property `kafka.topic.discovery.polling.interval` in order to
 periodically check for new topics into the Kafka cluster so the plugin will be able
-to send events to the defined topics.
\ No newline at end of file
+to send events to the defined topics.
+
+==== Multi Database Support
+
+Neo4j 4.0 Enterprise has https://neo4j.com/docs/operations-manual/4.0/manage-databases/[multi-tenancy support],
+in order to support this feature you can set for each database instance a configuration suffix with the following pattern
+`from.<DB_NAME>` to the properties in your neo4j.conf file.
+
+Following the list of new properties that allows to support multi-tenancy:
+
+----
+streams.source.topic.nodes.<TOPIC_NAME>.from.<DB_NAME>=<PATTERN>
+streams.source.topic.relationships.<TOPIC_NAME>.from.<DB_NAME>=<PATTERN>
+streams.source.enabled.from.<DB_NAME>=<true/false, default=true>
+streams.procedures.enabled.<DB_NAME>=<true/false, default=true>
+----
+
+This means that for each db instance you can specify if:
+* use the source connector
+* the routing patterns
+* use the procedures
+
+So if you have a instance name `foo` you can specify a configuration in this way:
+
+----
+streams.source.topic.nodes.myTopic.from.foo=<PATTERN>
+streams.source.topic.relationships.myTopic.from.foo=<PATTERN>
+streams.source.enabled.from.foo=<true/false, default=true>
+streams.procedures.enabled.foo=<true/false, default=true>
+----
+
+The old properties:
+
+----
+streams.source.enabled=<true/false, default=true>
+streams.source.topic.nodes.<TOPIC_NAME>=<PATTERN>
+streams.source.topic.relationships.<TOPIC_NAME>=<PATTERN>
+streams.procedures.enabled=<true/false, default=true>
+----
+
+are still valid and they refer to Neo4j's default db instance.
```

Comment:
Add note: the default database is controlled by Neo4j's `dbms.default_database` configuration property so we're being clear about which default database applies for this user.